### ISSUE TITLE
[Guild market] Buy and offer the same item again as one-click operation

### ISSUE THREAD

**Issue by drzuby** [DinoDevs/GladiatusCrazyAddon#461]
Many guilds use guild market for gold packaging in the market using items being continuously bought and offered again with the same price.  This requires entering the guild market, buying the appropriate item, then going to the packages, extraction of the item to the inventory, then going to the market again and offering the item with the same price at it was bought.

The improvement could be adding separate button in each row with offers to buy & offer again. There are few obstacles to be overcome but from technical perspective it's achievable.

**Comment by FrutyX** [comment]
I can see Apo already added a **feature request** label to it already, however, I would just like to state two things:

- This might act as a bot, because it plays/does multiple actions for you, getting the addon blacklisted
- Not to mention that when it does multiple actions at the same time, it creates strain on the server (we have seen in the past that Gladiatus servers can have problems when you are too fast), getting the addon blacklisted

**Comment by GreatApo** [comment]
> I can see Apo already added a **feature request** label to it already, however, I would just like to state two things:
> 
> - This might act as a bot, because it plays/does multiple actions for you, getting the addon blacklisted
> - Not to mention that when it does multiple actions at the same time, it creates strain on the server (we have seen in the past that Gladiatus servers can have problems when you are too fast), getting the addon blacklisted
> 

I don't think it is that bad and it could potentially save some server load. However, I don't know when it might be implemented.

**Comment by GramThanos** [comment]
I am against such features, behavior in the grey bot-like area. May give advantage Could potentially blame the addon for server load (it happened in the past).

So...
Yes, it is a problem... but I think GameForge should do something for that.

**Comment by GreatApo** [comment]
> I am against such features, behavior in the grey bot-like area. May give advantage Could potentially blame the addon for server load (it happened in the past).
> 
> So...
> Yes, it is a problem... but I think GameForge should do something for that.

It's a simple action for one item, no big deal 🤷🏻

**Comment by FrutyX** [comment]
I gotta side with Thanos here, there are features that should be implemented directly by the game developers, not by GCA.

The last time I saw a feature making multiple requests at the same time, it was only throwing errors... #424, #425.
